packages/lit-dev-content/site/tutorials/content/async-directive/03.md

```diff
@@ -0,0 +1,39 @@
+Before we make our directive update asynchronously, first let's get a 3rd-party
```

Comment:
Although this was mentioned in the intro, it doesn't seem like there's other context for it yet, so maybe omit leading phrase?

Reply:
Done